src/components/icon/utils.ts

```diff
@@ -58,24 +58,32 @@ export async function fetchIcon({ icon, scale }: FetchIconProps): Promise<Calcit
  * Normalize the icon name to match the path data module exports.
  * Exported for testing purposes.
  *
- * @param name
+ * @param name – an icon name that can be either kebab or camel-cased
  * @private
  */
 export function normalizeIconName(name: string): string {
   const numberLeadingName = !isNaN(Number(name.charAt(0)));
   const parts = name.split("-");
+  const kebabCased = parts.length > 0;
 
-  if (parts.length === 1) {
-    return numberLeadingName ? `i${name}` : name;
-  }
+  if (kebabCased) {
+    const firstNonDigitInPartPattern = /[a-z]/i;
+
+    name = name
+      .split("-")
```

Comment:
We also already split `name` into `parts` on line 66, so we can use that here instead of splitting again?

Reply:
Doh! Excellent catch! Fixing!